web_src/less/_base.less

```diff
@@ -853,6 +853,12 @@ a.commit-statuses-trigger {
   color: var(--color-text);
 }
 
+.ui.list > .item > img.ui.avatar + .content {
+  display: inline-block;
+  vertical-align: top;
+  padding-left: .5em;
+}
+
```

Comment:
This selector seems too complex. If there is a certain place using the avatar, do the helper CSS classes help?

Reply:
If it is a original Fomantic selector, I'm happy to keep it, even if complex. It'll ensure compatibility.

Reply:
Actually it is not an original Fomantic selector, but there is a [similar selector](https://github.com/go-gitea/gitea/blob/e7ef94e00f1319e5fb876f47fee28728bd671f07/web_src/fomantic/build/semantic.css#L28272) and I replaced `.image` with `img.ui.avatar`.

Reply:
Which page and which elements will this style be applied to?

Reply:
According to my test, this style is applied to the `explore org` page? If so, you could just update the `img.ui.avatar` styles in `_explore.less`, no need to introduce a new complex CSS selector.

Reply:
It will be applied in the user list of `/explore/users` page.
<img src="https://user-images.githubusercontent.com/15528715/219870858-39e23488-54e7-4925-b311-997aa79a98a7.png" width="600px"/>

Reply:
This is good enough:

![image](https://user-images.githubusercontent.com/2114189/219871311-c91dc331-e0a9-4978-a73b-504b22fa0eb1.png)


```
diff --git a/web_src/less/_explore.less b/web_src/less/_explore.less
index 5a41f8dc5..5caf21d41 100644
--- a/web_src/less/_explore.less
+++ b/web_src/less/_explore.less
@@ -69,6 +69,7 @@
 .ui.user.list {
   .item {
     padding-bottom: 25px;
+    display: flex;
 
     &:not(:first-child) {
       border-top: 1px solid var(--color-secondary);
@@ -78,6 +79,7 @@
     img.ui.avatar {
       width: 40px;
       height: 40px;
+      margin-right: 10px;
     }
 
     .description {
```

Reply:
> This is good enough:
> 
> ```diff
> diff --git a/web_src/less/_explore.less b/web_src/less/_explore.less
> index 5a41f8dc5..5caf21d41 100644
> --- a/web_src/less/_explore.less
> +++ b/web_src/less/_explore.less
> @@ -69,6 +69,7 @@
>  .ui.user.list {
>    .item {
>      padding-bottom: 25px;
> +    display: flex;
>  
>      &:not(:first-child) {
>        border-top: 1px solid var(--color-secondary);
> @@ -78,6 +79,7 @@
>      img.ui.avatar {
>        width: 40px;
>        height: 40px;
> +      margin-right: 10px;
>      }
>  
>      .description {
> ```
> 

Good idea. Thank you very much!